### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreia-oca**

### CODE REVIEW

File: playground/README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ As any playground - we have a few rules:
 * Add content and commit messages in english
 * Try to add descriptive commit messages
 * Be nice with any other content you find in here
+* Have fun
```
Comment (andreia-oca) [inline playground/README.md:10]: Nice

Comment (Alex-deVis) [reply playground/README.md:10]: sad af

Comment (andreia-oca) [reply playground/README.md:10]: Review is not clear - how do you propose to fix this?

Comment (Alex-deVis) [reply playground/README.md:10]: Having a beer is always a good start. Except for driving, #stay-safe.

**Pull request closed without merging**